WalletWasabi/BitcoinCore/Processes/BitcoindRpcProcessBridge.cs

```diff
@@ -60,7 +60,7 @@ public async Task StartAsync(CancellationToken cancel)
 
 				if (process is null || process.HasExited)
 				{
-					throw ex;
+					throw new Exception($"Failed to start daemon, location: '{process?.StartInfo.FileName} {process?.StartInfo.Arguments}'", ex);
```

Comment:
Make it more specific exception.

Reply:
`HttpRequestException`?

Reply:
No.

Reply:
Is there an StartupException type (perhaps create one)? otherwise `Exception` type is used in many places in the code (which is fine) for consistency.